lib/graphql/schema/loader.rb

```diff
@@ -102,7 +102,7 @@ def define_type(type, type_resolver)
               name: type["name"],
               type: type_resolver.call(type["type"]),
               description: type["description"],
-              default_value: type["defaultValue"]
+              default_value: type["defaultValue"] ? JSON.parse(type["defaultValue"], quirks_mode: true) : nil
```

Comment:
Oh huh, so is this a "how did it ever work" moment? Because previously the default value would _always_ have been a string, but with this commit, it's properly turned back into a float, int, boolean or, if it's a string, the extra quotes are removed? 

👏 👏 ! I think a test case for that behavior would be great, that way we don't um, re-break it in the future!